omnibus/config/projects/agent.rb

```diff
@@ -181,3 +181,9 @@
 
 exclude '\.git*'
 exclude 'bundler\/git'
+
+# removing the man pages from the embedded folder to reduce package size by ~4MB
+if not windows? # linux + osx
+  FileUtils.rm_f("#{install_dir}/embedded/man")
+  FileUtils.rm_f("#{install_dir}/embedded/share/man")
+end
```

Comment:
This cannot work, as `omnibus-ruby` creates the package in the following order:
- the project file (this file) is evaluated ; all dependencies declared in the project file also have their files evaluated,
- each of the declared dependencies is downloaded,
- the declared `build` blocks in all evaluated files are executed (and put their results in the installation directory).

This means that your `rm_f` commands are processed before anything is present in the installation directory, thus they do nothing 😅.

We have a "software definition" dedicated to cleaning up the package after installing everything else: `omnibus/config/software/datadog-agent-finalize.rb`, I think the manpages cleanup should belong there.